app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -186,13 +186,13 @@ class TabTrayView(
             tabsAdapter.tabTrayInteractor = interactor
             tabsAdapter.onTabsUpdated = {
                 if (view.context.settings().gridTabView) {
-                    concatAdapter.addAdapter(collectionsButtonAdapter)
                     concatAdapter.addAdapter(syncedTabsController.adapter)
+                    concatAdapter.addAdapter(collectionsButtonAdapter)
                 } else {
-                    // Put the 'Add to collections' button after the tabs have loaded.
-                    concatAdapter.addAdapter(0, collectionsButtonAdapter)
-                    // Put the Synced Tabs adapter at the end.
+                    // Synced Tabs adapter
                     concatAdapter.addAdapter(syncedTabsController.adapter)
+                    // Put the 'Add to collections' button after the tabs have loaded.
+                    concatAdapter.addAdapter(collectionsButtonAdapter)
```

Comment:
```suggestion
                    concatAdapter.addAdapter(syncedTabsController.adapter)
                    concatAdapter.addAdapter(collectionsButtonAdapter)
```
I think we might as well remove the comments here since it is self explanatory.